stock_cancel/stock.py

```diff
@@ -49,6 +49,10 @@ def action_revert_done(self):
                     _('Picking %s has invoices!') % (picking.name))
             picking.move_lines.write({'state': 'draft'})
             picking.state = 'draft'
+            for move in picking.move_lines:
+                for quant in picking.move_lines.quant_ids:
```

Comment:
I think you want `for quant in move.quant_ids` otherwise it would crash on a picking with 2 lines.